monitor/services/src/main/java/org/datacleaner/monitor/server/components/ComponentHandlerFactory.java

```diff
@@ -77,4 +88,23 @@ private void initialize() {
             analysisListener = new CompositeAnalysisListener(listeners.toArray(new AnalysisListener[listeners.size()]));
         }
     }
+
+    public ComponentDescriptor<?> resolveDescriptor(DataCleanerEnvironment env, String componentName) {
+        ComponentDescriptor<?> descriptor = env.getDescriptorProvider()
+                .getTransformerDescriptorByDisplayName(componentName);
+        if (descriptor == null) {
```

Comment:
Yuck, maybe we should add a new method on ```DescriptorProvider``` like ```getComponentDescriptorByDisplayName(...)```. It should then check for all component types (incl. Filters which are missing here).